### PR TITLE
28- Eu payroll

### PR DESCRIPTION
**Pull request by euhoro**
# Overview

## Summary
add glue jobs for payroll
resolve https://github.com/safebooks-ai/engine/issues/28

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## How has this been tested? 
Tested flows end-to-end from customer on dev env. Everything worked as expected:

please see https://github.com/safebooks-ai/engine/pull/173 for more evidence

## Checklist
- [x] My pull request title follows the format `[<jira-ticket/github-issue>]: <short-description>`
- [x] My code satisfies the jira-ticket's / github issue's acceptance criteria
- [x] My code follows the code style set for the project
- [x] I have made corresponding changes to the documentation
- [x] I have added at least one reviewer for this PR

### CODE REVIEW

**Comment by euhoro** [comment]
image

**Review by schammah — DISMISSED** [review]

File: tf-common/glue_jobs.tfvars [hunk 1]
```diff
@@ -75,5 +75,169 @@ glue_jobs = [
...
+    rds_connection  = true
+  },
+  {
+    name            = "rl_pr_attendance_reconciliation_payroll_time_management",
+    script_location = "/engine/rl_pr/rl_pr_attendance_reconciliation_payroll_time_management/rl_pr_attendance_reconciliation_payroll_time_management.py",
+    rds_connection  = true
+  },
+  {
+    name            = "l_pr_overhead_persent_change-socialSecurity",
+    script_location = "/engine/rl_pr/l_pr_overhead_persent_change-socialSecurity/l_pr_overhead_persent_change-socialSecurity.py",
```
Comment (schammah) [inline tf-common/glue_jobs.tfvars:240]: fix typo

Comment (euhoro) [reply tf-common/glue_jobs.tfvars]: fixed

File: tf-common/glue_jobs.tfvars [hunk 2]
```diff
@@ -75,5 +75,169 @@ glue_jobs = [
...
+    rds_connection  = true
+  },
+  {
+    name            = "rl_pr_salary_change",
+    script_location = "/engine/rl_pr/rl_pr_salary_change/rl_pr_salary_change.py",
+    rds_connection  = true
+  },
+  {
+    name            = "rl_pr_Recurring_one_time_payment",
+    script_location = "/engine/rl_pr/rl_pr_Recurring_one_time_payment/rl_pr_Recurring_one_time_payment.py",
```
[the remark this answers is not in the document]
Comment (euhoro) [reply tf-common/glue_jobs.tfvars:105]: the scope of this pr was to take existing data science work and make it available in prod. next step is to change all of these jupyter notebooks (30 at least )

**Review by schammah — APPROVED** [review]

**Pull request merged**